bldsys/cmake/template/sample_api/sample.cpp.in

```diff
@@ -0,0 +1,174 @@
+/* Copyright (c) 2023, Mobica Limited
```

Comment:
If sample will be generated with this license than I would suggest changing it to "Arm Limited and Contributors".
@Szymon-Rajczakowski-Mobica  what do you think?

```suggestion
/* Copyright (c) 2023, Arm Limited and Contributors
```